eventmesh-connectors/eventmesh-connector-file/src/main/resources/source-config.yml

```diff
@@ -28,13 +27,7 @@ connectorConfig:
     connectorName: fileSource
     nameserver: 127.0.0.1:9877
     topic: TopicTest
```

Comment:
`nameserver` and `topic` are actually redundant for this Connector.

Reply:
> `nameserver` and `topic` are actually redundant for this Connector.

Updated.